app/src/main/java/org/oppia/android/app/devoptions/ForceDownloadDialogFragment.kt

```diff
@@ -0,0 +1,34 @@
+package org.oppia.android.app.devoptions
+
+import android.app.Dialog
+import android.content.Context
+import android.os.Bundle
+import org.oppia.android.app.fragment.FragmentComponentImpl
+import org.oppia.android.app.fragment.InjectableDialogFragment
+import org.oppia.android.domain.devoptions.ForceDownloadParametersController
+import javax.inject.Inject
+
+/** Dialog fragment shown for force downloading remote parameters. */
+class ForceDownloadDialogFragment : InjectableDialogFragment() {
+  @Inject
+  lateinit var forceDownloadDialogFragmentPresenter:
+    ForceDownloadDialogFragmentPresenter
+  @Inject
+  lateinit var forceDownloadParametersController: ForceDownloadParametersController
+
+  companion object {
+    /** Returns a new instance of [ForceDownloadDialogFragment]. */
+    fun newInstance(): ForceDownloadDialogFragment =
+      ForceDownloadDialogFragment()
```

Comment:
Nit: Can be a single line?
```suggestion
    fun newInstance(): ForceDownloadDialogFragment = ForceDownloadDialogFragment()
```

Reply:
Now since we are using `ForceDownloadRemoteParameters` so it will require two lines.